### PR TITLE
Set keystone osp17 as non-voting

### PR DESCRIPTION
**Pull request by cescgina**
Until rhbz#2052499 is resolved.

### CODE REVIEW

File: znoyder/config.yml [hunk 1]
```diff
@@ -133,13 +133,6 @@ add:
           - 'check'
           - 'gate'
 
-  'keystone':
-    'osp-17.0':
-      'openstack-tox-py39':
-        type:
-          - 'check'
-          - 'gate'
-
```
Comment (sdatko) [inline znoyder/config.yml:142]: Please do not remove this – openstack-tox-py39 is not defined in upstream for Keystone (other projects have it as non-voting by default), so this is what this entry exists in add map.

**Review by sdatko — CHANGES_REQUESTED** [review]
The changes in override map are find, just please keep entry in add map.

File: znoyder/config.yml (hunk 1 above)
Comment (cescgina) [reply znoyder/config.yml:142]: Right, I'll add it back, thanks!

**Review by sdatko — APPROVED** [review]

File: znoyder/config.yml (hunk 1 above)
Comment (sdatko) [reply znoyder/config.yml]: Thanks, now it is great :-) :+1:

**Pull request merged**